### PR TITLE
ant passed by lint and fixed

### PR DESCRIPTION
**Pull request by MatiasVerdugo**

### CODE REVIEW

File: katas/langtonant/matias/main.test.js
Comment (J0c4) [inline katas/langtonant/matias/main.test.js:1]: This is a good start. Please add more tests for verifying all the functionality you have here

File: katas/langtonant/matias/main.js
Comment (J0c4) [inline katas/langtonant/matias/main.js:1]: Looks like implementation still in progress, please let me know if you are planning to push more changes to complete the Kata

**Review by J0c4 — CHANGES_REQUESTED** [review]

**Comment by J0c4** [comment]
There are some failures when running test and lint, please check and fix them along with fixing merge conflicts

**Review by J0c4 — CHANGES_REQUESTED** [review]
Hi Matias, your implementation looks good. Few observations:
- You need to add more test for smaller pieces in your code, that way, you verify that smaller functions are verified in multiple scenarios
- I was not able to test it using the main file, did I miss something?

Also, apparently, there is a syntax error in the line 7 of main.js file that is detected by lint, that needs to be fixed before we merge this branch, please fix and we can merge this

**Comment by MatiasVerdugo** [comment]
> Hi Matias, your implementation looks good. Few observations:
> 
> * You need to add more test for smaller pieces in your code, that way, you verify that smaller functions are verified in multiple scenarios
> * I was not able to test it using the main file, did I miss something?
> 
> Also, apparently, there is a syntax error in the line 7 of main.js file that is detected by lint, that needs to be fixed before we merge this branch, please fix and we can merge this

I've fixed the error on the line 7 updating the parser of lint, and the error was gone.

**Comment by J0c4** [comment]
> > Hi Matias, your implementation looks good. Few observations:
> > 
> > * You need to add more test for smaller pieces in your code, that way, you verify that smaller functions are verified in multiple scenarios
> > * I was not able to test it using the main file, did I miss something?
> > 
> > Also, apparently, there is a syntax error in the line 7 of main.js file that is detected by lint, that needs to be fixed before we merge this branch, please fix and we can merge this
> 
> I've fixed the error on the line 7 updating the parser of lint, and the error was gone.

Looks like issue still present, variable does not seem to be correctly declared

**Comment by J0c4** [comment]
Hi @MatiasVerdugo , any updates on this one? if lint issues do not get fixed, we cannot merge this as it will break the build status. Since its been a while for this PR, we will need to set tomorrow as a deadline so we can close it.

**Comment by MatiasVerdugo** [comment]
> Hi @MatiasVerdugo , any updates on this one? if lint issues do not get fixed, we cannot merge this as it will break the build status. Since its been a while for this PR, we will need to set tomorrow as a deadline so we can close it.

I'm sorry for the delay is already updated

**Comment by MatiasVerdugo** [comment]
> > Hi Matias, your implementation looks good. Few observations:
> > 
> > * You need to add more test for smaller pieces in your code, that way, you verify that smaller functions are verified in multiple scenarios
> > * I was not able to test it using the main file, did I miss something?
> > 
> > Also, apparently, there is a syntax error in the line 7 of main.js file that is detected by lint, that needs to be fixed before we merge this branch, please fix and we can merge this
> 
> I've fixed the error on the line 7 updating the parser of lint, and the error was gone.

"Unexpected token errors in ESLint parsing occur due to incompatibility between your development environment and ESLint's current parsing capabilities with the ongoing changes with JavaScripts ES6~7."

The solution is to have ESLint parsed by a compatible parser, i.e @babel/eslint-parser or babel-eslint for babel version below v7.

just add:

"parser": "@babel/eslint-parser"
to your .eslintrc file and run npm install @babel/eslint-parser --save-dev or yarn add -D @babel/eslint-parser.

**Comment by J0c4** [comment]
hi @MatiasVerdugo , unfortunately there is a missing change that is adding eslint-parser to package.json which is added by running the command npm install @babel/eslint-parser that you have in your comment. Without this, the configuration that you added will not work.

As mentioned, lets close this one for now without merging. We can follow up if you have questions about this topic

**Pull request closed without merging**
